Jenkinsfile

```
@@ -124,6 +124,20 @@ pipeline {
                         }
                     }
                 }
+                stage('distroVersion() tests') {
+                    steps {
+                        withEnv(['BRANCH_NAME=release2.4']) {
```

The branch name is `release/2.4`.

```suggestion
                        withEnv(['BRANCH_NAME=release/2.4']) {
```

It probably doesn't make a material difference to the test, but should be accurate at least for appearances sake.